Lido-Script/script.js

```diff
@@ -60,7 +60,7 @@ const signer = wallet.connect(null);
 
 // Setup SDK for mainnet, make sure to change withdrawal credentials and fee recipient if you change the network/chainId!
 const client = new Client(
-  { baseUrl: "https://api.obol.tech", chainId: 1 },
+  { baseUrl: "https://api.obol.tech", chainId: 17000 },
```

Comment:
should the lido script have an argument in it for mainnet/holesky instead of this switching?